src/components/course/course-header/CoursePreview.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
 import { PlayCircleFilled } from '@edx/paragon/icons';
 import { useToggle, Image } from '@edx/paragon';
-import { VideoPlayer } from '../../video';
+
+import VideoPlayer from '../../video/VideoPlayer';
```

Comment:
[curious/clarification] Why migrate away from the named import to the default import here? Could we keep using the named export?